packages/remix-react/routeModules.ts

```diff
@@ -9,7 +10,8 @@ import type {
 import type { AppData } from "./data";
 import type { LinkDescriptor } from "./links";
 import type { EntryRoute } from "./routes";
-import type { RouteData } from "./routeData";
+
+type RouteData = RouterState["loaderData"];
```

Comment:
Grab this right from the router now instead of re-defining the `string->any` map